Cargo.toml

```diff
@@ -1,3 +1,11 @@
+[workspace.package]
```

Comment:
According to the [docs](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-package-table), `repository` and `license` are not supported for workspace inheritance.

Reply:
The docs say that they are supported right? They're both there in the table.

Reply:
I don't see them in the supported list

```
<p>Keys that are supported:</p>
<div class="table-wrapper">

  |  
-- | --
authors | categories
description | documentation
edition | exclude
homepage | include
keywords | license
license-file | publish
readme | repository
rust-version | version

</div>
```